docs_examples/tutorial/create_metrics_app.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import boto3
+
+from aws_lambda_powertools import Logger, Tracer
+from aws_lambda_powertools.event_handler import APIGatewayRestResolver
+from aws_lambda_powertools.logging import correlation_paths
+
+cold_start = True
+metric_namespace = "MyApp"
+
+logger = Logger(service="APP")
+tracer = Tracer(service="APP")
+metrics = boto3.client("cloudwatch")
+app = APIGatewayRestResolver()
+
+
+@tracer.capture_method
+def add_greeting_metric(service: str = "APP"):
+    function_name = os.getenv("AWS_LAMBDA_FUNCTION_NAME", "undefined")
+    service_dimension = {"Name": "service", "Value": service}
+    function_dimension = {"Name": "function_name", "Value": function_name}
+    is_cold_start = True
+
+    global cold_start
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Global variables are dangerous because they can be simultaneously accessed from multiple sections of a program. This frequently results in bugs. Most bugs involving global variables arise from one function reading and acting on the value of a global variable before another function has the chance to set it to an appropriate value. We recommend using a configuration module to mutate global state.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules)

Reply:
@heitorlessa lol. this is funny now that CodeGuru is picking up the code in the docs now ;)